log_example_test.go

```diff
@@ -394,3 +421,19 @@ func ExampleContext_Durs() {
 
 	// Output: {"foo":"bar","durs":[10000,20000],"message":"hello world"}
 }
+
+func ExampleFatal() {
+	err := errors.New("A repo man spends his life getting into tense situations")
+	service := "myservice"
+
+	zerolog.TimeFieldFormat = ""
+
+	log.Fatal().
+		Err(err).
+		Str("service", service).
+		Msgf("Cannot start %s", service)
+
+	// Produces: {"time":1516133263,"level":"fatal","error":"A repo man spends his life getting into tense situations","service":"myservice","message":"Cannot start myservice"}
+	//         exit status 1
+
```

Comment:
Unneeded `exit status 1` and extra blank line.

Reply:
dropped